concrete/views/panels/details/page/location.php

```diff
@@ -118,6 +118,32 @@
 
 $(function() {
 
+    $('button[data-dialog-action="submit"]').on('click', function(e) {
+        e.preventDefault();
+        let formData = $('form[data-dialog-form="location"]').serialize();
+
+        $.ajax({
+            type: 'POST',
+            url: '<?= $controller->action('check') ?>',
```

Comment:
When sending strings from PHP to JavaScript, I prefer using `json_encode()`: it's safer...

```php
 url: <?= json_encode((string) $controller->action('check')) ?>,